odl_video/settings.py

```diff
@@ -440,11 +445,15 @@ def get_all_config_keys():
     '1351620000001-200010,1351620000001-200020,1351620000001-200050'
 ).split(',')
 
+if ET_PRESET_IDS == ['']:  # This may happen if `ET_PRESET_IDS=` is in .env file.
+    raise ValueError('ET_PRESET_IDS cannot be blank, please check your settings & environment')
```

Comment:
probably [ImproperlyConfigured](https://docs.djangoproject.com/en/1.11/ref/exceptions/#improperlyconfigured) is better here